lib/cli/commands/validate.js

```diff
@@ -16,47 +15,54 @@ module.exports = function (opts) {
     },
     handler: async function (argv) {
       let userPackageJson
-      let packageJsonPath
       try {
-        const jsonFile = path.join(process.cwd(), 'package.json')
-        packageJsonPath = path.dirname(jsonFile)
-        userPackageJson = await fs.readJSON(jsonFile)
+        userPackageJson = await fs.readJSON(path.join(process.cwd(), 'package.json'))
       } catch (error) {
         argv.log.error(error)
         return
       }
 
-      const ajv = new Ajv({
-        // coerceTypes: true,
-        useDefaults: true,
-        removeAdditional: true,
-        allErrors: true,
-        nullable: true
-      })
-
-      let { support } = userPackageJson
-
-      if (support === true) {
-        support = await fs.readJSON(path.join(packageJsonPath, 'package-support.json'))
-      } else if (typeof support === 'string') {
-        const p = path.join(packageJsonPath, support)
-        if (!pathContains(packageJsonPath, p)) {
-          throw new Error('Support string path must be contained in cwd')
+      let supportData = userPackageJson.support
+      if ((supportData === false) || supportData) {
+        // first validate that the supportData is correct
+        if ((supportData !== true) && !(supportData instanceof Object)) {
+          console.log(errorInSupportElement)
+          console.log('Support element should either be true or an Object matching the required schema')
+          return
```

Comment:
nitpick: `else` is redundant with early returns, prefer reduced nesting where possible.

Reply:
will fix